src/main/java/teammates/logic/core/FeedbackQuestionsLogic.java

```diff
@@ -357,6 +373,14 @@ public int compare(FeedbackQuestionAttributes q1, FeedbackQuestionAttributes q2)
         
         FeedbackParticipantType recipientType = question.recipientType;
         
+        String giverTeam = getTeamNameForUser(giver, instructorGiver, studentGiver);
+        
+        addRecipientsBasedOnRecipientType(question, giver, studentGiver, recipients, recipientType, giverTeam);
```

Comment:
Looks like you don't need the `recipients` parameter at all, it's always an empty map when it's passed in?